conf/compose/docker-compose.nginx.yml

```diff
@@ -5,7 +5,6 @@ services:
     volumes:
       - ${CAMINO_HOME}/conf/nginx/nginx.conf:/etc/nginx/nginx.conf
       - ${CAMINO_HOME}/conf/nginx/templates/default.conf.template:/etc/nginx/templates/default.conf.template
-      - ${CAMINO_HOME}/conf/uwsgi/uwsgi_params:/etc/nginx/uwsgi_params
```

Comment:
Can we add this back to the core and cms ymls?

Reply:
Ahh, good catch I mistook the ini for the params.